commit 0243957c4f48095037b33f1ab73c8322a69df74c
Author: Hannes Reinecke <hare@suse.de>
Date:   Fri Feb 17 09:02:45 2017 +0100

    scsi: use 'scsi_device_from_queue()' for scsi_dh
    
    commit 857de6e00778738dc3d61f75acbac35bdc48e533 upstream.
    
    The device handler needs to check if a given queue belongs to a scsi
    device; only then does it make sense to attach a device handler.
    
    [mkp: dropped flags]
    
    Signed-off-by: Hannes Reinecke <hare@suse.com>
    Reviewed-by: Christoph Hellwig <hch@lst.de>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1270_0243957c4f48095037b33f1ab73c8322a69df74c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1270_0243957c4f48095037b33f1ab73c8322a69df74c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
